lib/facter/clientcert_pem.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add(:clientcert_pem) do
+  setcode do
```

Comment:
If this is a concern being always-on, we can set it to opt-in by adding an escape hatch here.

```Ruby
next unless File.exist? '/etc/puppetlabs/node_encrypt_fact'
```